### PR TITLE
Updated News and Winging It Descriptions

### PR DESCRIPTION
**Pull request by SondraE**
## Related Issue(s)
[101](https://github.com/orgs/oddbird/projects/13/views/1?pane=issue&itemId=47726747)

## Steps to test/reproduce
/wingingit/ - Description
/oddnews/ - Description, bullet points, and footer
(Also https://www.youtube.com/@wingingitoddbird)

### CODE REVIEW

File: content/oddnews.njk [hunk 1]
```diff
@@ -3,9 +3,12 @@ title: OddNews
 sub: Because best practice is a conversation...
 banner: OddNews
 summary: |
-  **Hi, OddFriends!** This is the newsletter for designers
-  & developers who want to **get the latest news in your field**
-  — UX Design, CSS, JavaScript or Python — and keep tabs on the full stack.
+  **Hi, OddFriends!**
+  This is the newsletter for designers & developers
+  who want the **latest news in front-end web development**.
+  Articles, videos, updates, and workshops
```
Comment (jgerigmeyer) [inline content/oddnews.njk:9]:
```suggestion
  who want the **latest news in front-end web development** --
  articles, videos, updates, and workshops
```

File: content/oddnews.njk [hunk 2]
```diff
@@ -21,12 +24,13 @@ summary: |
 {%- set title = 'What to Expect' -%}
 {%- set contents -%}
   Here are a few examples of topics featured in past editions of OddNews.
-  - **Bird of the Month** - Introduction to the oddest of birds
   - **CSS** - [Miriam Suzanne's](/authors/miriam/) latest work in Container Queries
```
Comment (jgerigmeyer) [inline content/oddnews.njk:27]:
```suggestion
  - **CSS** - [Miriam Suzanne's](/authors/miriam/) latest work on Container Queries
```

File: content/oddnews.njk [hunk 3]
```diff
@@ -21,12 +24,13 @@ summary: |
...
-  - **Python** - How to Run Playwright Inside Docker
   - **Sass** - How to use the new `split()` function in Sass
-  - **Custom App Development** - Web Apps vs. PWAs vs. Native
-  - **UX Design** - How to conenct designers and developers with Object Oriented UX
+  - **JavaScript** - Guide to the new Node.js package importer
```
Comment (jgerigmeyer) [inline content/oddnews.njk:29]:
```suggestion
  - **JavaScript** - Guide to Sass's new Node.js package importer
```

File: content/oddnews.njk [hunk 4]
```diff
@@ -21,12 +24,13 @@ summary: |
...
+  - **App Development** - Web Apps vs. PWAs vs. Native
+  - **UX Design** - Strategies for designing more intuitive websites
+  - **Workshop** - Register for [Cascading Layouts CSS Workshop](https://www.oddbird.net/workshops/cascading-layouts/)
```
Comment (jgerigmeyer) [inline content/oddnews.njk:32]:
```suggestion
  - **Workshop** - Register for [Cascading Layouts CSS Workshop](/workshops/cascading-layouts/)
```

File: content/oddnews.njk [hunk 5]
```diff
@@ -21,12 +24,13 @@ summary: |
...
 {%- set contents -%}
   Here are a few examples of topics featured in past editions of OddNews.
-  - **Bird of the Month** - Introduction to the oddest of birds
   - **CSS** - [Miriam Suzanne's](/authors/miriam/) latest work in Container Queries
-  - **Python** - How to Run Playwright Inside Docker
   - **Sass** - How to use the new `split()` function in Sass
-  - **Custom App Development** - Web Apps vs. PWAs vs. Native
-  - **UX Design** - How to conenct designers and developers with Object Oriented UX
+  - **JavaScript** - Guide to the new Node.js package importer
+  - **App Development** - Web Apps vs. PWAs vs. Native
```
Comment (jgerigmeyer) [inline content/oddnews.njk:30]:
```suggestion
  - **App Development** - Web apps vs. PWAs vs. Native apps
```

File: content/wingingit.md [hunk 6]
```diff
@@ -4,12 +4,18 @@ index: Winging It
...
 summary: |
-  Hi, OddFriends! At OddBird, we enjoy connecting and having conversations
-  across disciplines. Join us as we *wing it* and discuss the latest in UX
-  Design, CSS, JavaScript, Python, and everything in between.
+  Hi, OddFriends!
+  Join OddBird
+  for conversations
+  at the cutting edge
+  of front-end web development
+  – from CSS, HTML, and JavaScript
```
Comment (jgerigmeyer) [inline content/wingingit.md:17]:
```suggestion
  of front-end web development --
  from CSS, HTML, and JavaScript
```

**Review by jgerigmeyer — APPROVED** [review]

File: content/oddnews.njk [hunk 7]
```diff
@@ -21,12 +24,13 @@ summary: |
 {%- set title = 'What to Expect' -%}
 {%- set contents -%}
   Here are a few examples of topics featured in past editions of OddNews.
-  - **Bird of the Month** - Introduction to the oddest of birds
   - **CSS** - [Miriam Suzanne's](/authors/miriam/) latest work in Container Queries
-  - **Python** - How to Run Playwright Inside Docker
   - **Sass** - How to use the new `split()` function in Sass
```
Comment (stacyk) [inline content/oddnews.njk:28]: I wonder if we should drop the word "new" on a few of these so it is a bit more long-term if we don't want to update them frequently.

Comment (stacyk) [inline content/oddnews.njk:28]:
```suggestion
  - **Sass** - How to use the `split()` function in Sass
```

File: content/oddnews.njk (hunk 3 above)
Comment (stacyk) [inline content/oddnews.njk:29]:
```suggestion
  - **JavaScript** - Guide to Sass's Node.js package importer
```
edited @jgerigmeyer's suggestion. Maybe we want to keep new in though and check theses every few months?

**Review by stacyk — APPROVED** [review]
just a suggestion, otherwise looks good along with Jonny's recommendations.

**Pull request merged**